### PR TITLE
Almost succeeded in expanding omerc is tmerc / merc example

### PR DESCRIPTION
**Pull request by jidanni**
Oops, could someone adjust the example to make the answers (3, 4) match? Thanks!

### CODE REVIEW

**Comment by rouault** [comment]
omerc and tmerc match, but only on the sphere, not a non-spheric ellipsoid:

$ echo 12 55 | proj +proj=omerc +alpha=0 +R=6400000
766869.97	6209742.96
$ echo 12 55 | proj +proj=tmerc  +R=6400000
766869.97	6209742.96

I suspect the maths behind the ellipsoidal tmerc formulation are more sophisticated than the "ellipsoidization" of omerc. https://www.iogp.org/wp-content/uploads/2019/09/373-07-02.pdf page 59 paragraph 2 seems to suggest that there are several potential approaches to that, so this must explain it is quite expected that omerc ellipsoidal != tmerc ellipsoidal

**Review by jidanni — COMMENTED** [review]
OK, I added the sphere part.

File: docs/source/operations/projections/omerc.rst [hunk 1]
```diff
@@ -71,17 +71,20 @@ In the second case, the azimuth is given indirectly by specifying two points on
...
-    1335833.89   7326837.71
-
+    $ echo 12 55 | proj +proj=merc
+    1335833.89	7326837.71
+    $ echo 12 55 | proj +proj=omerc +alpha=90
+    1335833.89	7326837.71
+    $ echo 12 55 | proj +proj=omerc +alpha=0 +R=6400000
+    766869.97 6209742.96
+    $ echo 12 55 | proj +proj=tmerc +R=6400000
+    766869.97 6209742.96
```
Comment (rouault) [inline docs/source/operations/projections/omerc.rst]:
```suggestion
    $ echo 12 55 | proj +proj=merc +ellps=GRS80
    1335833.89	7326837.71

    $ echo 12 55 | proj +proj=omerc +alpha=90 +ellps=GRS80
    1335833.89	7326837.71

    $ echo 12 55 | proj +proj=omerc +alpha=0 +R=6400000
    766869.97 6209742.96

    $ echo 12 55 | proj +proj=tmerc +R=6400000
    766869.97 6209742.96
```

File: docs/source/operations/projections/omerc.rst [hunk 2]
```diff
@@ -71,17 +71,23 @@ In the second case, the azimuth is given indirectly by specifying two points on
 the central line, using the options
 :option:`+lat_1`, :option:`+lon_1`, :option:`+lat_2`, and :option:`+lon_2`.
 
-Example: Verify that the Mercator projection is a limiting form of the Oblique
-Mercator
+Example: Verify that the Mercator, and Transverse Mercator (on a sphere),
+projectios are limiting forms of the Oblique Mercator
```
Comment (rouault) [inline docs/source/operations/projections/omerc.rst]:
```suggestion
projections are limiting forms of the Oblique Mercator
```

**Pull request merged**